### PR TITLE
Fix klockwork issue for type UNINIT.CTOR.MUST

### PR DESCRIPTION
**Pull request by lyjbupt**

### CODE REVIEW

File: omx_components/src/mfx_omx_vdec_component.cpp [hunk 1]
```diff
@@ -97,6 +97,7 @@ MfxOmxVdecComponent::MfxOmxVdecComponent(OMX_ERRORTYPE &error,
     m_pFreeSyncPoint(NULL),
     m_nLockedSurfacesNum(0),
     m_nCountDecodedFrames(0),
+    m_extBuffersi{},
```
Comment (TianmiChen) [inline omx_components/src/mfx_omx_vdec_component.cpp]: is it m_extBufersi(NULL)?

Thanks,

File: omx_components/src/mfx_omx_vpp_wrapp.cpp [hunk 2]
```diff
@@ -35,6 +35,7 @@ MfxOmxVppWrapp::MfxOmxVppWrapp(void):
     m_pVPP(NULL),
     m_session(NULL),
     m_numVppSurfaces(0),
+    m_EncodeCtrls{},
```
Comment (TianmiChen) [inline omx_components/src/mfx_omx_vpp_wrapp.cpp]: Is it m_EncodeCtrls(NULL)?

File: omx_utils/include/spl/mfx_omx_avc_structures.h [hunk 3]
```diff
@@ -909,6 +909,7 @@ struct AVCSEIPayLoad : public HeapObject, public AVCSEIPayLoadBase
 
     AVCSEIPayLoad()
         : AVCSEIPayLoadBase()
+	, user_data{}
```
Comment (TianmiChen) [inline omx_utils/include/spl/mfx_omx_avc_structures.h]: Same as above.

File: omx_utils/include/spl/mfx_omx_hevc_structures.h [hunk 4]
```diff
@@ -520,6 +520,7 @@ struct ReferencePictureSetList
 
     ReferencePictureSetList()
     : m_NumberOfReferencePictureSets(0)
+    , referencePictureSet{}
```
Comment (TianmiChen) [inline omx_utils/include/spl/mfx_omx_hevc_structures.h]: Same as above.

File: omx_utils/src/spl/mfx_omx_avc_bitstream.cpp [hunk 5]
```diff
@@ -253,6 +253,10 @@ inline void FillScalingList8x8(AVCScalingList8x8 *scl_dst,mfxU8 *coefs_src)
 }
 
 AVCBaseBitstream::AVCBaseBitstream()
+	: m_pbs(0)
+	, m_bitOffset(31)
+	, m_pbsBase(0)
+	, m_maxBsSize(0)
```
Comment (dmitrybrazhkin) [inline omx_utils/src/spl/mfx_omx_avc_bitstream.cpp]: Reset(0, 0) does the same. So please restore the code or remove Reset(0, 0)

**Pull request merged**